js/components/app.js

```diff
@@ -46,6 +46,10 @@ const DEFAULT_AUTHORED_STATE = {
   chartAvgs: false
 };
 
+function validAuthoredState(state) {
+  return state && state.columns
```

Comment:
Wouldn't it be something like this instead:
```suggestion
  return state && state.columns ? state : undefined
```

Reply:
Yes you are right. Thanks. :)